.expeditor/verify.pipeline.yml

```diff
@@ -934,38 +667,22 @@ steps:
       automatic:
         limit: 1
 
-  - label: "[build] :linux: aarch64 launcher"
-    agents:
-      queue: default-privileged-aarch64
+  - label: "[build] :linux: pkg-cfize"
```

Comment:
lets keep this build out since we discovered we no longer support it.

Reply:
I'm confident that I'm tracking but making sure, you want me to remove that build step from the pipeline? Makes sense, just having some compulsion to confirm this one.